### ISSUE TITLE
Chrome 48 removes SVGGraphicsElement.getTransformToElement()

### ISSUE THREAD

**Issue by gnome74699** [huei90/snap.svg.zpd#57]
The most recent version of chrome (48.0.2564.97) removes `zpdElement.node.getTransformToElement` which is used when creating edges.

Therefore using `snap.svg.zpd` in this latest beta version of Chrome throws errors like:
`TypeError: zpdElement.node.getTransformToElement is not a function(…)`

How best to fix this in `snap.svg.zpd`?

**Comment by huei90** [comment]
@tikiatua

**Comment by tikiatua** [comment]
Hmm..

Apparently the getTransformToElement should be back in the latest version. There does however seem to be a fix
```javascript
SVGElement.prototype.getTransformToElement = SVGElement.prototype.getTransformToElement || function(elem) {
    return elem.getScreenCTM().inverse().multiply(this.getScreenCTM());
};
```
More details here https://github.com/cpettitt/dagre-d3/issues/202

**Comment by gnome74699** [comment]
Ok thanks, guy, I'll try it.

**Comment by codeHatcher** [comment]
@tikiatua, just as a heads up, latest chrome mac Version 48.0.2564.103 (64-bit) doesn't have getTransformToElement

**Comment by dianchen96** [comment]
The latest 48.0.2564.109 (64-bit) version still does not fix this issue.

**Comment by tikiatua** [comment]
Hi there,

should be fixed in the latest version (v0.0.11)
